package.json

```diff
@@ -34,7 +34,7 @@
     "cross-env": "^7.0.2",
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.2.2",
-    "eslint": "8",
+    "eslint": "8.29.0",
```

Comment:
Note: this has been changed to a fixed version, which is the version we have been using, because of https://github.com/xojs/xo/issues/718